app/routes/about-us/about-us.module.scss

```diff
@@ -0,0 +1,74 @@
+.root {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    min-height: 85vh;
+    margin-bottom: 60px;
+    gap: 6px;
+}
+
+.aboutSection {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.header1 {
+    font: var(--paragraph1);
+    font-size: 22px;
```

Comment:
I think `font` declaration can be removed. You defined custom font-size, line-height and font-family are inherited from root.